services/should-show-out-of-france-banner.js

```diff
@@ -1,14 +1,43 @@
 import { config } from '~/config/environment'
 
 export async function shouldShowOutOfFranceBanner(baseUrl) {
-  const FR_COUNTRY_CODE = 'FR'
-  const url = new URL('/geolocate', baseUrl)
-
   if (!config.isFrenchDomain || !config.isPixSite) return false
 
+  const country = await _getUserCountryFromGeolocationService(baseUrl)
+  if (!country) return false
+
+  return _shouldShowBannerForCountry(country)
+}
+
+async function _getUserCountryFromGeolocationService(baseUrl) {
+  const url = new URL('/geolocate', baseUrl)
+
   const response = await fetch(url).then((res) => res.json())
+  if (!response) return undefined
 
-  if (!response?.country) return false
+  return response.country
+}
+
+function _shouldShowBannerForCountry(country) {
+  const FR_COUNTRY_CODE = 'FR'
+  const DOM_TOM_COUNTRY_CODES = [
+    'GA',
+    'GF',
+    'MQ',
+    'RE',
+    'YT',
+    'NC',
+    'PF',
+    'BL',
+    'MF',
+    'PM',
+    'TF',
+    'WF',
+  ]
+  const COUNTRIES_WHERE_BANNER_SHOULD_NOT_BE_SHOWN = [
+    FR_COUNTRY_CODE,
+    ...DOM_TOM_COUNTRY_CODES,
+  ]
```

Comment:
Les constantes gagneraient à être déclarées globalement à la racine du module de manière à n'être évaluées qu'une fois plutôt que chaque fois que la fonction `shouldShowBannerForCountry` est appelée :
```suggestion
const FR_COUNTRY_CODE = 'FR'
const DOM_TOM_COUNTRY_CODES = [
  'GA',
  'GF',
  'MQ',
  'RE',
  'YT',
  'NC',
  'PF',
  'BL',
  'MF',
  'PM',
  'TF',
  'WF',
]
const COUNTRIES_WHERE_BANNER_SHOULD_NOT_BE_SHOWN = [
  FR_COUNTRY_CODE,
  ...DOM_TOM_COUNTRY_CODES,
]

function _shouldShowBannerForCountry(country) {
```